gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilPlugin.kt

```diff
@@ -423,3 +307,98 @@ private fun <T : Task> T.doFirstCompat(block: (T) -> Unit) {
     }
   })
 }
+
+private inline fun <reified T : Task> Project.namedLazy(
```

Comment:
Is there something like this built in?

Reply:
Maybe ask trobalik?

Reply:
Asked him, there's no built-in solution.